android/java/org/chromium/chrome/browser/crypto_wallet/fragments/dapps/EncryptionKeyFragment.java

```diff
@@ -128,14 +129,17 @@ private void initState() {
                         encryptionPublicKeyRequest -> {
                             if (encryptionPublicKeyRequest != null) {
                                 mEncryptionPublicKeyRequest = encryptionPublicKeyRequest;
-                                SpannableStringBuilder requestDescription =
-                                        new SpannableStringBuilder();
-                                requestDescription.append(Utils.geteTLD(
-                                        encryptionPublicKeyRequest.originInfo.eTldPlusOne));
-                                requestDescription.append(" ");
-                                requestDescription.append(getString(
-                                        R.string.brave_wallet_provide_encryption_key_description));
-                                mTvMessageDesc.setText(requestDescription);
+                                String formattedeTLD = String.format(
+                                        getString(
+                                                R.string.brave_wallet_provide_encryption_key_description),
+                                        Utils.geteTLDHTMLFormatted(
+                                                encryptionPublicKeyRequest.originInfo.eTldPlusOne));
+                                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+                                    mTvMessageDesc.setText(Html.fromHtml(
+                                            formattedeTLD, Html.FROM_HTML_MODE_LEGACY));
+                                } else {
+                                    mTvMessageDesc.setText(Html.fromHtml(formattedeTLD));
+                                }
```

Comment:
nit: can create a helper for HTML conversion in AndroidUtils as it will be used from time to time.